services/simcore/docker-compose.yml

```diff
@@ -532,6 +532,34 @@ services:
           memory: 128M
           cpus: '0.1'
 
+  efs-guardian:
+    networks:
```

Comment:
Are you sure this service needs to be in the `public` network?

Reply:
Actually I am also sceptical now about the `monitoring` network, as SAN mentioned as well

Reply:
networks were removed :+1: